DEVELOPMENT.md

```diff
@@ -23,6 +23,18 @@ See JSON files in `tools/codegen/base` directory for examples of the manifest.
 > GITHUB_TOKEN=$(gh auth status --show-token 2>&1 | sed -n 's/^.*Token: \(.*\)$/\1/p') ./tools/manifest.sh <tool>
 > ```
 
+## Local testing
```

Comment:
`main.sh` is a script that is likely to change contants outside the repository, so I don't think it should be run locally, basically, although it would be fine if you are testing it within an isolated environment, such as a container.

Anyway, I don't want to require contributors to run tests locally.

Reply:
Ok, I've removed those additions